packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useTranslation } from '../../../utils';
+
+export interface IDevice {
+    id: number;
+    name: string;
+    country: string;
+    capacity: number;
+    imported?: boolean;
+}
+
+const useStyles = makeStyles({
+    device: {
+        background: '#373737',
+        marginBottom: '10px',
+        boxShadow: '0 2px 4px rgba(0,0,0,.2)'
+    },
+
+    header: {
+        padding: '12px 18px',
+        display: 'flex'
+    },
+
+    footer: {
+        padding: '12px 0 16px 0',
+        boxSizing: 'border-box',
+        background: '#3B3B3B',
+        display: 'flex',
+        justifyContent: 'space-evenly'
+    },
+
+    icon: {
+        width: 55,
+        height: 55,
+        color: '#969696',
+        background: '#969696',
+        marginRight: '22px'
+    },
+
+    label: {
+        fontSize: 12,
+        color: 'white',
+        opacity: 0.75
+    },
+
+    value: {
+        fontSize: 14,
+        color: 'white'
+    },
+
+    name: {
+        fontSize: 16,
+        color: 'white'
+    },
+
+    grow: {
+        flex: 1
+    },
+
+    button: {
+        borderColor: '#894EC5'
+    },
+
+    buttonLabel: {
+        color: '#894EC5',
+        textTransform: 'uppercase'
+    }
+});
+
```

Comment:
Hard-coded colors on styles above. 
Also, please, start the name of the file containing component with a capital letter.

Reply:
renamed